tests/foreman/ui/test_host.py

```diff
@@ -105,20 +107,82 @@ def setUpClass(cls):
         ).create()
         cls.loc_name = cls.loc.name
 
+        # Get the Partition table ID
+        cls.ptable = entities.PartitionTable().search(
+            query={
+                u'search': u'name="{0}"'.format(DEFAULT_PTABLE)
+            }
+        )[0]
+
+        # Get the OS ID
+        cls.os = entities.OperatingSystem().search(query={
+            u'search': u'name="RedHat" AND (major="{0}" OR major="{1}")'
+                       .format(RHEL_6_MAJOR_VERSION, RHEL_7_MAJOR_VERSION)
+        })[0]
+
+        # Get the templates and update with OS, Org, Location
+        cls.templates = []
+        for template_name in [
+            'Kickstart default PXELinux',
+            'Discovery Red Hat kexec',
+            'Kickstart default iPXE',
+            'Satellite Kickstart Default',
+            'Satellite Kickstart Default Finish',
+            'Satellite Kickstart Default User Data'
+        ]:
+            template = entities.ConfigTemplate().search(
+                query={
+                    u'search': u'name="{}"'.format(template_name)
+                }
+            )[0]
+            template.operatingsystem = [cls.os]
+            template.organization = [cls.org_]
+            template.location = [cls.loc]
```

Comment:
Are you sure this change is safe? I remember we had similar issues before. Maybe it's better to append existing lists of orgs/locs (in case `cls.org_` is not there already) not to break any other test?

Reply:
Oh, i see you simply moved this section completely. Ok, i don't mind if it was working ok before, it's up to you. Imo it's just a bit too dangerous to leave it as is - since it can introduce very random failure for other tests which will be very hard to debug and find a root cause.

Reply:
To be honest, I would be happy to break any other test to find isolation issues in case we have any